release-notes/major10/10.1.0/valtimo-backend-libraries.md

```diff
@@ -20,6 +20,11 @@ The following features were added:
   the id of a form flow instance. More
   information can be found [here](/reference/modules/form-flow.md#available-properties-in-spel-context)
 
+* **Form flow conditional next step**
+
+  A next step in a form flow can now be conditional. This allows for more complex form flows. More information can be
+  found [here](/using-valtimo/form-flow/create-form-flow-definition.md)
```

Comment:
Maybe also mention the submission data type change here again with a reason as to why the change is made (e.g. to make it possible to directly access fields of the submission data in expressions)?